internal/engine/metrics.go

```diff
@@ -85,27 +85,27 @@ func (e *ExecutorMetrics) CountEvalStatus(
 // CountRemediationStatus counts remediation events by status.
 func (e *ExecutorMetrics) CountRemediationStatus(
 	ctx context.Context,
-	status string,
+	status db.RemediationStatusTypes,
 	evalID uuid.UUID,
 	projectID uuid.UUID,
 ) {
 	e.evalCounter.Add(ctx, 1, metric.WithAttributes(
 		attribute.String("profile_id", evalID.String()),
 		attribute.String("project_id", projectID.String()),
```

Comment:
You probably _don't_ want to expose these attributes as metric tags, because they will blow up your [metric cardinality](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#cardinality-limits).

Generally, since metrics are aggregated in-memory, you don't want to put non-enum attributes into a metric.